### PR TITLE
Configs

### LINKED ISSUE #4: Support more configutations
Re-enable configuration support for non-local execution.

See for instance [tensorspdz](https://github.com/mortendahl/privateml/tree/master/tensorflow/spdz/configs).

### PR DESCRIPTION
**Pull request by mortendahl**
Closes #4

### CODE REVIEW

File: src/example-server.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import sys
```
Comment (jvmancuso) [inline src/example-server.py]: should this file be moved to the examples folder?

**Comment by mortendahl** [comment]
experimenting with expanding this to also handle input providers and output receivers

File: tensorflow_encrypted/tests/conv2d_test.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import unittest
 from tensorflow_encrypted.protocol import Pond
-from tensorflow_encrypted.protocol import Server
+from tensorflow_encrypted.protocol import Player
```
Comment (morgangiraud) [inline tensorflow_encrypted/tests/conv2d_test.py]: Maybe refactor those imports as one? like
```
from tensorflow_encrypted.protocol import (
    Pond
    Player
)
```

File: examples/logreg.py [hunk 3]
```diff
@@ -49,9 +49,9 @@ def generate_fake_training_data():
     FakeInputProvider('/job:localhost/replica:0/task:0/device:CPU:5')
 ]
 
-server0 = tfe.protocol.Server('/job:localhost/replica:0/task:0/device:CPU:0')
-server1 = tfe.protocol.Server('/job:localhost/replica:0/task:0/device:CPU:1')
-crypto_producer = tfe.protocol.CryptoProducer('/job:localhost/replica:0/task:0/device:CPU:2')
+server0 = tfe.protocol.Player('/job:localhost/replica:0/task:0/device:CPU:0')
```
Comment (morgangiraud) [inline examples/logreg.py]: You can probably refactor all the example files

File: tensorflow_encrypted/protocol/pond.py [hunk 4]
```diff
@@ -391,7 +390,7 @@ def sigmoid(self, x):
 
         return z
 
-    def reveal(prot, x):
+    def reveal(self, x):
```
Comment (morgangiraud) [inline tensorflow_encrypted/protocol/pond.py]: 👍

File: tests/conv2d_test.py [hunk 5]
```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import unittest
 from tensorflow_encrypted.protocol import Pond
```
Comment (morgangiraud) [inline tests/conv2d_test.py]: use Multi import statement
`from tensorflow_encrypted.protocol import Player, Pond`

File: examples/logreg.py (hunk 3 above)
Comment (mortendahl) [reply examples/logreg.py]: done

**Pull request merged**